### PR TITLE
feat(origin-device-registry-local-irec-api): capacity is string now

### PR DESCRIPTION
**Pull request by ioncreature**

### CODE REVIEW

File: packages/devices/origin-device-registry-irec-local-api/src/device/device.service.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export class DeviceService {
 
         const irecDevice = await this.irecService.createDevice(user, {
             ...deviceData,
-            capacity: deviceData.capacity / 1e6,
+            capacity: String(Number(deviceData.capacity) / 1e6),
```
Comment (josipbagaric) [inline packages/devices/origin-device-registry-irec-local-api/src/device/device.service.ts]: `Number(deviceData.capacity)` will overflow if the number is too big.

We should use BigNumber here.
```js
BigNumber.from(deviceData.capacity).div(1e6).toString()
```

**Review by josipbagaric — CHANGES_REQUESTED** [review]

File: packages/devices/origin-device-registry-irec-local-api/migrations/1631576581265-CapacityString.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CapacityString1631576581265 implements MigrationInterface {
+    name = 'CapacityString1631576581265';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `ALTER TABLE "irec_device_registry_device" ALTER COLUMN "capacity" TYPE character varying`
+        );
+    }
```
Comment (josipbagaric) [inline packages/devices/origin-device-registry-irec-local-api/migrations/1631576581265-CapacityString.ts:10]: Please create a migration for existing device capacities, it should be simple.

File: packages/devices/origin-device-registry-irec-local-api/src/device/device.service.ts (hunk 1 above)
Comment (ioncreature) [reply packages/devices/origin-device-registry-irec-local-api/src/device/device.service.ts:69]: Number.MAX_SAFE_INTEGER is 9007199254740991, I think it is enough

File: packages/devices/origin-device-registry-irec-local-api/migrations/1631576581265-CapacityString.ts (hunk 2 above)
Comment (ioncreature) [reply packages/devices/origin-device-registry-irec-local-api/migrations/1631576581265-CapacityString.ts:10]: Altering types do not delete data, it changes int to string preserving all the data we had before

File: packages/devices/origin-device-registry-irec-local-api/src/device/device.service.ts (hunk 1 above)
Comment (ioncreature) [reply packages/devices/origin-device-registry-irec-local-api/src/device/device.service.ts]: fixed

**Review by alexworker23 — APPROVED** [review]

**Review by josipbagaric — APPROVED** [review]

**Pull request merged**